populate_db.py

```diff
@@ -0,0 +1,107 @@
+import pyphen, requests, json
+#pyphen.language_fallback('ipa_en')
+ipa_dic = pyphen.Pyphen(lang='ipa_en')
+spelling_dic = pyphen.Pyphen(lang='en')
+
+URL = "http://127.0.0.1:5000"
+LANGUAGE = "english"
+
+
+print(ipa_dic.inserted("sʌbdʒɛktɪv"))  # use dark magic space trick in the future
+
+wordlist = "allprons.txt"
```

Comment:
global variables are usually not a good idea. Maybe put that in a function. and use the `if __name__ === "__main__"` pattern to prevent from side effects at import